src/components/Search/index.tsx

```diff
@@ -197,10 +204,18 @@ const Results = connectStateResults(
   }
 )
 
-const Search = ({ handleSearchSelect, useKeyboardShortcuts }) => {
+interface ISearchProps {
+  handleSearchSelect: () => void
+  useKeyboardShortcuts: boolean
+}
+
+const Search: React.FC<ISearchProps> = ({
+  handleSearchSelect,
+  useKeyboardShortcuts,
+}) => {
   const intl = useIntl()
-  const containerRef = useRef()
-  const inputRef = useRef()
+  const containerRef = useRef<HTMLDivElement>(null)
```

Comment:
A Ref needs to be explicitly initiated with `null` otherwise, it will lead to a TypeError as `undefined` is not a part of the type definition for a Mutuable RefObject